sort/js/search.js

```diff
@@ -0,0 +1,62 @@
+var RANDOM_STEP = 8;
+
+function _nextRandom() {
```

Comment:
these function should be moved from global scope.
could you try to use namespace pattern? it was described previous lesson.